mappings/net/minecraft/SharedConstants.mapping

```diff
@@ -18,8 +18,11 @@ CLASS net/minecraft/class_155 net/minecraft/SharedConstants
 	FIELD field_29734 RELEASE_TARGET Ljava/lang/String;
 	FIELD field_29735 RELEASE_TARGET_PROTOCOL_VERSION I
 	FIELD field_29737 SNBT_TOO_OLD_THRESHOLD I
+	FIELD field_29738 RESOURCE_PACK_VERSION I
+	FIELD field_29739 DATA_PACK_VERSION I
 	FIELD field_29740 DATA_VERSION_KEY Ljava/lang/String;
 	FIELD field_34371 DEBUG_BIOME_SOURCE Z
+	FIELD field_34373 MAIN_SERIES Ljava/lang/String;
```

Comment:
is this the current or the main series? need to check exp as well

Reply:
It doesn't seem like field_34373 exists in exp. There's another constant with value `ccpreview`, though.

Reply:
If in exp8 there is no static final string field with `main` value, then I would believe this field is the current series field. Notice that the intermediaries are generated anew from 1.17.1 than based off exp snapshots, so there is no intermediary correspondence (cannot find `field_34373` in exps).
```suggestion
	FIELD field_34373 CURRENT_SERIES Ljava/lang/String;
```